tree-sitter-go/tree-sitter-go.cabal

```diff
@@ -14,16 +14,16 @@ build-type:          Simple
 library
   exposed-modules:     TreeSitter.Go
   build-depends:       base                 >= 4.12 && < 5
-                     , tree-sitter >= 0.1 && < 0.3
+                     , tree-sitter          == 0.3.0.0
                      , template-haskell     >= 2.12.0.0 && < 2.15.0.0
                      , tree-sitter-go-internal
   default-language:    Haskell2010
 
 library tree-sitter-go-internal
   exposed-modules:     TreeSitter.Go.Internal
   hs-source-dirs:      internal
-  build-depends:       base                 >= 4.12 && < 5
-                     , tree-sitter >= 0.1 && < 0.3
+  build-depends:       base
+                     , tree-sitter
```

Comment:
Internal libraries can’t be used outside of the package they’re defined in, and any constraints they impose are effectively unioned with the public library, so I’ve just deleted them all and allowed the public library components to constrain things instead.